src/main/kotlin/com/wynntils/athena/routes/AuthenticationRoutes.kt

```diff
@@ -42,25 +43,29 @@ class AuthenticationRoutes {
      */
     @Route(path = "/responseEncryption", type = RouteType.POST)
     fun responseEncryption(ctx: Context): JSONObject {
+        generalLog.info("Calling /responseEncryption")
         val response = JSONObject()
         val body = ctx.body().asJSON<JSONObject>()
 
         if (!body.containsKey("username") || !body.containsKey("key") || !body.containsKey("version")) {
```

Comment:
You probably want to use const val keys for these instead of writing strings each time, seeing as they've been used around code quite often

Reply:
I'd also argue to do this in a separate pr